CONSTITUTION.md

```diff
@@ -164,13 +165,13 @@ In the absence of both the President and the Vice-President, the Committee membe
 
 - 7.2.1. The Annual General Meeting shall be held in the month of October each year.
 
-- 7.2.2. The quorum at Annual General Meeting shall be fifty percent (50%) of members entitled to vote or fourteen (14) members entitled to vote whichever is lesser.
```

Comment:
But I thought ProgSoc membership was at an all-time high what with free memberships and whathaveyou! Accordingly, I can't see how you would struggle to make quorum in this day and age. 

Historically, the quorum was 20.

Reply:
Just would've been more of a thought in case of issue and running subsequent AGM, last year had 18 members present at the AGM, year before 20. The membership numbers don't seem to reflect interest in the AGM. Wouldn't see a major issue considering activate has 6 as there template, or perhaps just explicitly in the case of a failed AGM, pushing it down to 12?

Reply:
@tomchristmas 
> But I thought ProgSoc membership was at an all-time high what with free memberships and whathaveyou!

This is true, but it seems people are less actively involved if they don't feel they have to get their $5 worth. We are likely to re-introduce the fee at $2-3 next year - partly too because FEIT is not subsidising as much as they make it seem.

> Accordingly, I can't see how you would struggle to make quorum in this day and age.

Last year we were looking at ~12-13 attending in the lead up to AGM afair, and asked around to encourage more people to show up. I think other clubs have a similar issue where mostly only people considering Exec roles attend, which isn't great for democratically picking the best candidate.

> Historically, the quorum was 20.

Definitely a far better member-exec ratio. 12 is pushing the limit of Execs voting themselves in. Personally I'm more in favour of keeping 14, but this has been raised a few times internally so it's good to have debate on it at an AGM.

As @MitchellKij said ActivateUTS sets the mandatory quorum at 6, far too low imo, but I think shows the struggle most clubs have to fill AGMs.

Reply:
I agree that the more votes the better. I think low attendance might be due to: 
1. People don't understand what an AGM is because they're not very involved in clubs or not executives. The idea of an AGM can be scary or a mystery to them.
2. People might think AGMs are not for regular/general members to attend. They might feel out of place, especially if they've never been involved in any of the club events.
3. People might not be able to attend at the designated time.
4. People are simply not interested in the prospects of the club or in voting, which may indicate a need to enhance member engagement and demonstrate the value of participation.

We don't know for sure which one is causing low attendance, but I think we agree that we want to increase attendance at AGMs. Points 1 and 2 can be addressed through clearer communication, publicity, and transparency. Point 3 can be addressed by making votes asynchronous and easy to do online (proxy votes don't work because most people aren't that invested to DM strangers or execs; at worst, this is tribalism, aside from the incentives for them probably not being too apparent). Introducing incentives for participation or highlighting the benefits of involvement might help to address point 4.

The suggestion to solve point 3 is a bit unconventional, but if we're really serious, we could record the AGM and delay the election decisions for a few days to accommodate the voting process. At the very least, we're certainly able to address the other points.

Reply:
I'd be more inclined to have a clause for a subsequent AGM being lowered to 12 rather than the standard, or just drop it.

I think most people are aware of what an AGM is, but there is no incentive for people to go most of the time, especially if they don't want a role, even food as an incentive isn't much, considering the duration unless someone is genuinely interested in it, I'd see no reason general members would attend.

I don't think we can delay election decisions, it seems a bit dodgy, possibly, depending on how it's done, as to what would be in place. Can people not present at the AGM vote after? If not, why couldn't they vote on the time period? that would be my thinking. Would just have to be done carefully and transparently (plus probably run this idea through activate as reporting all this would be interesting depending on approach as the minutes/ reporting wouldn't be done on the day/ elections of positions).

Reply:
If the only incentive for people to go is only to get a role, it makes sense for us to expect a low number since there can be only so many executives. I think making this assumption doesn't consider the bigger picture of why clubs exist though, which is because people are interested in the business of the club. This doesn't necessarily mean that only people who are executives are interested in the business of the club.

Delaying election decisions would indeed require transparency and a trustable voting process/platform. We could consider this for future AGMs by implementing it ourselves with ClubVotingSystem or other alternatives, if not for this year's.

Reply:
> Can people not present at the AGM vote after?

Standard practice for any kind of voting is to have a decision making window that starts from the time the meeting is announced and the motions are published for your consideration, to the conclusion of the meeting. 

It is during this window that questions can be asked, which btw is what we are doing **right now** by commenting on these pull requests, and votes can be cast in absentia.

Reply:
> proxy votes don't work

I (and others) have acted as a proxy at ProgSoc AGMs in the past, but only because the Execs of the day allowed it for those meetings. In each instance, they were Alumni who were unable to attend but still cared enough and trusted me to vote in the best interests of the club.Certainly not randoms on Discord.

So this Exec will need to decide - and declare - whether or not proxies will be allowed this year.